packages/core/src/blocks/ImageBlockContent/ImageBlockContent.ts

```diff
@@ -70,7 +70,23 @@ export const imageRender = (
     image.src = block.props.url;
   }
 
-  image.alt = block.props.name || block.props.caption || "BlockNote image";
+  // Accessibility: set alt/aria based on presence of caption per RGAA
+  const accessibleImageWithCaption = () => {
+    image.alt = block.props.caption;
+    image.removeAttribute("aria-hidden");
+    image.removeAttribute("role");
+    image.setAttribute("tabindex", "0");
+  };
+
+  const accessibleImage = () => {
+    image.alt = "";
+    image.setAttribute("role", "presentation");
+    image.setAttribute("aria-hidden", "true");
+    image.setAttribute("tabindex", "-1");
+  };
+
+  block.props.caption ? accessibleImageWithCaption() : accessibleImage();
```

Comment:
Same as below, just fallback when there is no alt, and this doesn't have the `block.props.name`